lib/hooks.php

```diff
@@ -1,5 +1,13 @@
 <?php namespace Evermade\Swiss\Hooks;
 
+function set_blog_public_option() {
```

Comment:
Neat! Do we want to use WordPress or PSR naming conventions here (and elsewhere)?